styles/main.css

```diff
@@ -773,3 +773,10 @@ button {
 .hide-component {
   display: none;
 }
+
```

Comment:
If you don't want to display an element, simply use the .hide-component convenience class.